composer/algorithms/augmix/augmix.py

```diff
@@ -207,13 +210,13 @@ class AugMix(Algorithm):
             version of each image. Default: ``1.0``.
         augmentation_set (str, optional): Must be one of the following options:
```

Comment:
It might be worth linking to `composer.algorithms.utils.augmentation_primitives.augmentation_sets` ([see it in the docs](https://mosaicml-composer--963.com.readthedocs.build/en/963/api_reference/composer.algorithms.utils.augmentation_primitives.html#composer.algorithms.utils.augmentation_primitives.augmentation_sets)) and in the corresponding argument docstring section in `RandAugment`.

Reply:
@growlix Great, I cross-linked between those two sections of the docs! @hanlint This PR is ready to merge.